### PR TITLE
New blog post

### PR DESCRIPTION
**Pull request by Adammatthiesen**

### CODE REVIEW

File: www/web/src/content/blog/whats-next-for-studiocms.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Whats next for StudioCMS?
+description: New development team member, and dashboard project!
+publishDate: 2024-10-03
+ogVariant: fall
+author: adam-matthiesen
+---
+## What's Next for StudioCMS?
```
Comment (TheOtterlord) [inline www/web/src/content/blog/whats-next-for-studiocms.md:8]:
```suggestion
```

File: www/web/src/content/blog/whats-next-for-studiocms.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Whats next for StudioCMS?
```
Comment (TheOtterlord) [inline www/web/src/content/blog/whats-next-for-studiocms.md:2]:
```suggestion
title: Welcoming our latest maintainer!
```

File: www/web/src/content/blog/whats-next-for-studiocms.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+description: New development team member, and dashboard project!
```
Comment (TheOtterlord) [inline www/web/src/content/blog/whats-next-for-studiocms.md:3]:
```suggestion
description: Welcoming our newest member, and paving the road to our dashboard overhaul.
```

File: www/web/src/content/blog/whats-next-for-studiocms.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+publishDate: 2024-10-03
+ogVariant: fall
+author: adam-matthiesen
+---
+## What's Next for StudioCMS?
+
+Astro Studio is sunsetting, we’ve updated our branding, and we’re proud to announce our first sponsorship thanks to Turso. But what’s on the horizon for StudioCMS? We invite you to join us as we continue improving and expanding our CMS to meet the needs of our growing community.
```
Comment (TheOtterlord) [inline www/web/src/content/blog/whats-next-for-studiocms.md:10]:
```suggestion
```

File: www/web/src/content/blog/whats-next-for-studiocms.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: Welcoming our latest maintainer!
+description: Welcoming our newest member, and paving the road to our dashboard overhaul.
```
Comment (TheOtterlord) [inline www/web/src/content/blog/whats-next-for-studiocms.md:3]:
```suggestion
description: Welcoming our newest member, and dashboard!
```

**Review by dreyfus92 — APPROVED** [review]
Welcome aboard @louisescher 🥳

**Review by TheOtterlord — APPROVED** [review]

**Pull request merged**